cclib/parser/psi3parser.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Parser for Psi3 and Psi4 output files"""
+
+
+import re
+
+import numpy
+
+from cclib.parser import data
+from cclib.parser import logfileparser
+from cclib.parser import utils
+
+
+class Psi3(logfileparser.Logfile):
+    """A Psi3 log file."""
+
+    def __init__(self, *args, **kwargs):
```

Comment:
If this does nothing except call the superclass, is it needed at all?

Reply:
I don't understand. We always need to call the superclass.

Reply:
I don't think so, the child class will inherit __init__ from its parent.

Consider this:
```python
class A(object):
  def __init__(self, input):
    print "In A:", input

class B(A):
  pass

A(1)
B(2)
```

This will print as expected, without `__init__` defined in `B`:
```
1
2
```

Reply:
It doesn't seem to be working:
```
********* SUMMARY OF ERRORS *********

basicPsi3/dvb_sp_hf.out: Customized restricted single point HF/KS unittest
testaooverlaps (test.data.testSP.Psi3SPTest)Traceback (most recent call last):
  File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  [Previous line repeated 1 more times]
  File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 184, in testaooverlaps
    self.assertEquals(self.data.aooverlaps.shape, (self.data.nbasis, self.data.nbasis))
  File "/home/eric/development/cclib_berquist/test/test_data.py", line 119, in ccdata_getattribute_with_coverage                                                                             
    return object.__getattribute__(self, attr)
AttributeError: 'ccData' object has no attribute 'aooverlaps'


********* SUMMARY OF FAILURES *********

basicPsi3/dvb_sp_hf.out: Customized restricted single point HF/KS unittest
testmetadata (test.data.testSP.Psi3SPTest)Traceback (most recent call last):
  File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
    testfunc(self, *args, **kwargs)
  [Previous line repeated 5 more times]
  File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 271, in testmetadata
    self.assertIn("basis_set", self.data.metadata)
AssertionError: 'basis_set' not found in {'package': 'Log', 'methods': [], 'success': False, 'package_version': 3}
```

Reply:
OK, that is a little weird. Must be a byproduct of the awkward way we have unit tests and skipping cases implemented. OK, we don't need to figure it out for this.